wenet/transformer/ctc.py

```diff
@@ -67,3 +69,67 @@ def argmax(self, hs_pad: torch.Tensor) -> torch.Tensor:
             torch.Tensor: argmax applied 2d tensor (B, Tmax)
         """
         return torch.argmax(self.ctc_lo(hs_pad), dim=2)
+
+    def forced_align(self,
+                     ctc_probs: torch.Tensor,
+                     y: torch.Tensor,
+                     blank_id=0) -> list:
+        """ctc forced alignment.
+
+        Args:
+            torch.Tensor ctc_probs: hidden state sequence, 2d tensor (T, D)
+            torch.Tensor y: id sequence tensor 1d tensor (L)
+            int blank_id: blank symbol index
+        Returns:
+            torch.Tensor: alignment result
+        """
+
+        ctc_hyps = ctc_probs
+
+        y_insert_blank = insert_blank(y, blank_id)
```

Comment:
maybe ys with comment is okay instead of y_insert_blank, which will make small code